backend/src/endpoints/users/users.controller.ts

```diff
@@ -114,7 +114,7 @@ export default class UsersController {
       validated_on: "-1",
       validation_method: validation_method,
       validation_value: validation_value,
-      progress_lesson: 0,
+      progress_lesson: 1,
```

Comment:
Either both need to be zero, or both need to be 1. Either is okay, but we want to be consistent.

Reply:
Updated progress_lesson to 0.